src/data/faq_de.json

```diff
@@ -102,7 +102,8 @@
                         "anchor": "access-list",
                         "active": true,
                         "textblock": [
-                            "Einige Firewalls verhindern möglicherweise den Zugriff der CWA auf das Backend. Das kann in einigen Fällen zu einer Fehlermeldung führen, unter anderem 'URSACHE: 4000, error during web request, HTTP status 901'. Sollten sie eine Firewall benutzen (Router oder App), vergewissern sie sich, dass die folgenden Domains in der Access-Liste für Web- und den DNS-Requests berücksichtigt sind:<br/>*.coronawarn.app<br/>*.t-online.de"
+                            "Einige Firewalleinstellungen verhindern möglicherweise den Zugriff der CWA auf das Corona-Warn-App-Backend. Das kann in einigen Fällen zu einer Fehlermeldung führen, unter anderem 'URSACHE: 4000, error during web request, HTTP status 901'. Sollten sie eine Firewall benutzen (Router oder App), vergewissern sie sich, dass die folgenden Domains in der Access-Liste für Web- und den DNS-Requests berücksichtigt sind:",

```

Comment:
In "Sollten **sie** eine Firewall benutzen (Router oder App), vergewissern **sie** sich, dass die folgenden Domains in der Access-Liste für Web- und den DNS-Requests berücksichtigt sind:"

change both occurrences of "sie" to "Sie".

Reply:
fixed